src/Header/About/index.js

```diff
@@ -0,0 +1,13 @@
+import './style.css'
```

Comment:
you can give one line space after importing file and starting code and format code